website/models.py

```diff
@@ -120,10 +120,20 @@ def validate_image(fieldfile_obj):
 
 
 class Issue(models.Model):
+    labels = (
+        (1, 'Number Error'),
+        (2, 'Functional'),
+        (3, 'Performance'),
+        (4, 'Security'),
+        (5, 'Typo'),
+        (6, 'Design'),
+        (0, 'Unspecified')
```

Comment:
How about changing *Unspecified* to *General* ? I think, that will cover some of the bugs which doesn't comes under the predefined category.